scripts/pause_menu.gd

```diff
@@ -0,0 +1,48 @@
+extends CanvasLayer
+
+func _ready() -> void:
+	if !get_parent().PROCESS_MODE_PAUSABLE:
+		print("warning: parent is not pausable")
+	init_mute_audio_button()
+
+func _input(_event: InputEvent) -> void:
```

Comment:
These work even while the pause menu is not brought up. Is this intended? I think they should only work when the pause menu is up